R/AlgorithmHandler.r

```diff
@@ -56,9 +56,10 @@ AlgorithmHandler <- methods::setRefClass(
         }
       }
       # Begin startup
-      outputFile <- fifo("/tmp/algoout", open="w", blocking=TRUE)
-      inputFile <- file(pipeName)
+      outputFile <- fifo("/tmp/algoout", blocking=TRUE)
+      inputFile <- file("stdin")
```

Comment:
isn't this supposed to be pipeName for testing purposes?

Reply:
yes, good catch - thanks updating.